src/utils/getTextForLanguage.js

```diff
@@ -24,23 +26,20 @@ const mergeCustomizer = function (objValue, srcValue) {
  * doesn't exist, it warns the coder and returns English.
  */
 const getTextForLanguage = function (langName) {
+  if (langName === 'en') {
+    return interpolatedEnglish;
 
-  let snippetsTemplate;
-  if (localizations[ langName ]) {
-
+  } else if (localizations[ langName ]) {
+    const interpolatedSnippets = interpolateSnippets(localizations[ langName ], inlineComponents);
+    
     // deeply merge the object containing snippets in langName with English,
     // so that we fall back to English if a particular field is missing.
-    snippetsTemplate = mergeWith({}, english, localizations[ langName ], mergeCustomizer);
+    return mergeWith({}, interpolatedEnglish, interpolatedSnippets, mergeCustomizer);
 
   } else {
-
     console.warn('There\'s no localization for ' + langName + '. Defaulting to English.');
-    snippetsTemplate = english;
-
+    return interpolatedEnglish;
```

Comment:
It's a bit confusing that we're returning something called 'interpolatedEnglish'. That's a wording issue, though, and we just tweak it in a different PR.

Reply:
Oh, my bad - this only happens if there isn't a matching language. Nvm.